x-pack/plugins/ml/public/application/components/job_spaces_sync/job_spaces_sync_flyout.tsx

```diff
@@ -153,13 +153,15 @@ export const JobSpacesSyncFlyout: FC<Props> = ({ onClose }) => {
 function getResponseCounts(resp: SyncSavedObjectResponse) {
   let successCount = 0;
   let errorCount = 0;
-  Object.values(resp).forEach((result: SavedObjectResult) => {
-    Object.values(result).forEach(({ success, error }) => {
-      if (success === true) {
-        successCount++;
-      } else if (error !== undefined) {
-        errorCount++;
-      }
+  Object.values(resp).forEach((result: SyncResult) => {
```

Comment:
As discussed, after synchronization is complete, the Overview page doesn't refresh so the warning callout remains. Is it possible to force a refresh, as is happening on the Trained Models page.

![image](https://user-images.githubusercontent.com/7405507/155119615-2eeb7fa1-9caa-4edd-bc80-7ccb197507c2.png)